integration-tests/tests/update_test.go

```diff
@@ -73,7 +73,8 @@ func (s *updateSuite) TestUpdateToSameReleaseAndChannel(c *check.C) {
 		common.Reboot(c)
 	} else if common.AfterReboot(c) {
 		common.RemoveRebootMark(c)
-		c.Assert(common.GetCurrentUbuntuCoreVersion(c) > common.GetSavedVersion(c),
-			check.Equals, true)
+		currentVersion := common.GetCurrentUbuntuCoreVersion(c)
+		c.Assert(currentVersion > common.GetSavedVersion(c), check.Equals, true,
```

Comment:
We adopted the style of always using Assert, probably because it looks more like python. We are never using Fatalf. 
Do you see a problem with that? I prefer the asserts, but I can make the change easily if there's a good reason.